compass/ocean/tests/global_ocean/mesh/arm10to60/namelist.split_explicit

```diff
@@ -0,0 +1,9 @@
+config_time_integrator = 'split_explicit'
+config_dt = '00:12:00'
+config_btr_dt = '00:00:30'
```

Comment:
@milenaveneziani and @darincomeau, these time steps need to be adjusted.  They are from the WC14.

Reply:
I commented on the individual lines but it's probably easier to keep it in this comment: `config_dt = '00:10:00'`, and we currently have `config_btr_dt = '0000_00:00:24'`, which may be a mistake?